src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventAttachments/groupEventAttachmentsFilter.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import omit from 'lodash/omit';
+import xor from 'lodash/xor';
+import {Link, withRouter} from 'react-router';
+import {WithRouterProps} from 'react-router/lib/withRouter';
+import styled from 'react-emotion';
+
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+
+const GroupEventAttachmentsFilter = (props: WithRouterProps) => {
+  const {query, pathname} = props.location;
+  const {types} = query;
+  const onlyCrashReportTypes = ['event.minidump', 'event.applecrashreport'];
+  const allAttachmentsQuery = omit(query, 'types');
+  const onlyCrashReportsQuery = {
+    ...query,
+    types: onlyCrashReportTypes,
+  };
+
+  return (
+    <FilterWrapper>
+      <div className="btn-group">
+        <Link
```

Comment:
You could use the `Button` component here instead of `Link` with class names. We're trying to not add more usage of styles provided by LESS so that we can remove them in the future.

Reply:
Great to know! Thank you, fixed.